composer.json

```diff
@@ -25,10 +25,7 @@
     "autoload": {
         "psr-4": {
             "Spatie\\TwitterStreamingApi\\": "src"
-        },
-        "classmap": [
```

Comment:
You were right, this line has to be removed, and it'll just work in both the project / package.